### ISSUE TITLE
Struggle to get it work with Exponent

### ISSUE THREAD

**Issue by melalj** [invertase/react-native-firebase-starter#2]
I still get the error with the starter

```
RNFirebase core module was not found natively on iOS, ensure you have correctly included the RNFirebase pod in your projects `Podfile` and have tun `pod install`.
```

Is it compatible with Exponent or need to run it natively?

**Comment by Ehesp** [comment]
Have you ejected? I

**Comment by melalj** [comment]
No, did not ejected. Should I?

**Comment by Ehesp** [comment]
This is a starter repo created with `react-native init` - how have you tried to install it?

**Comment by melalj** [comment]
react-native-cli: `2.0.1`
react-native: `0.48.4`
node: `8.7.0`
yarn: `1.2.1`

I followed the README to try the starter kit but it fails:
```sh
git clone https://github.com/invertase/react-native-firebase-starter.git
cd react-native-firebase-starter
yarn install
```

When I install the pods (`cd ios && pod install && cd ../`) with the PodFile you provided: it fails right away (when it tries to install BoringSSL): `sed: -e expression #1, char 1: unknown command: `.'`.
So I commented out all optional packages and installed pods properly.

Now when I run `react-native run-ios`.
I get this error: `Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/Build/Products/Debug-iphonesimulator/RNFirebaseStarter.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist`

Am I missing something?
